apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryInsertRowTest.cs

```diff
@@ -63,22 +63,40 @@ public void ToRowsData_NoSpecifiedInsertId()
             };
             Assert.Equal("value1", row["field1"]);
             row["field3"] = 2;
-            var rowData = row.ToRowsData();
+            var rowData = row.ToRowsData(false);
             Assert.Equal("value1", rowData.Json["field1"]);
             Assert.Null(rowData.Json["field2"]);
             Assert.Equal(2, rowData.Json["field3"]);
             // The insert ID should be populated automatically if not supplied by the user.
             Assert.NotNull(rowData.InsertId);
         }
 
+        [Fact]
+        public void ToRowsData_NoSpecifiedInsertId_EmptyInsertIdAllowed()
+        {
+            var row = new BigQueryInsertRow
+            {
+                { "field1", "value1" },
+                { "field2", null }
+            };
+            Assert.Equal("value1", row["field1"]);
+            row["field3"] = 2;
+            var rowData = row.ToRowsData(true);
+            Assert.Equal("value1", rowData.Json["field1"]);
+            Assert.Null(rowData.Json["field2"]);
+            Assert.Equal(2, rowData.Json["field3"]);
+            // The insert ID should be populated automatically if not supplied by the user.
```

Comment:
This comment isn't accurate for this test :)

Reply:
Done!